addon/manifest.json

```diff
@@ -2,7 +2,7 @@
   "description": "A tool for Mozillians who want to know how many reviews are in their queue in their browser.",
   "manifest_version": 2,
   "name": "MyQOnly",
-  "version": "0.7.1",
+  "version": "0.8.0",
```

Comment:
If you wouldn't mind reverting the version number bump here and in package.json, I'll do that in a separate commit with release notes, etc.